### PR TITLE
Support for season type

### PR DESCRIPTION
**Pull request by sunkast**
Added support for season type items and thumbnail fallback to parentThumb if grandparentThumb does not exist

### CODE REVIEW

File: MMM-plex-recently-added.js [hunk 1]
```diff
@@ -102,8 +102,13 @@ Module.register('MMM-plex-recently-added', {
 
   _getThumbUrl(item) {
     var thumbKey = 'thumb';
-    if (item.type === 'episode') {
+    if (item.type === 'episode' | item.type === 'season') {
```
Comment (Gyran) [inline MMM-plex-recently-added.js]: Should probably be `||`? Otherwise it's a bitwise OR

File: MMM-plex-recently-added.js [hunk 2]
```diff
@@ -102,8 +102,13 @@ Module.register('MMM-plex-recently-added', {
...
+		if (item.grandparentThumb){
```
Comment (Gyran) [inline MMM-plex-recently-added.js:106]: Do we want the `grandparentThumb` to take precedence over the `parentThumb`? 

could it be something like:
```
var thumb;
if (item.parentThumb) {
	thumb = item.parentThumb;
} else if (item. grandparentThumb) {
	thumb = item.grandparentThumb;
}
```

or the other way around?

**Review by Gyran — COMMENTED** [review]
Nice work, thanks!

File: MMM-plex-recently-added.js (hunk 2 above)
Comment (sunkast) [reply MMM-plex-recently-added.js:106]: grandparentThumb can take precedence. Just need to check for parentThumb if grandparentThumb does not exist.

File: MMM-plex-recently-added.js (hunk 1 above)
Comment (sunkast) [reply MMM-plex-recently-added.js]: Yes. You are correct.

**Pull request merged**